site/en/gemma/docs/jax_inference.ipynb

```diff
@@ -181,50 +181,860 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 3,
+      "execution_count": null,
       "metadata": {
         "id": "_W3FUd9lt8VT"
       },
       "outputs": [],
       "source": [
-        "GEMMA_VARIANT = '2b-it' # @param ['2b', '2b-it'] {type:\"string\"}"
+        "GEMMA_VARIANT = 'gemma2-2b-it' # @param ['gemma2-2b', 'gemma2-2b-it'] {type:\"string\"}"
       ]
     },
     {
       "cell_type": "code",
-      "execution_count": 4,
+      "execution_count": null,
       "metadata": {
-        "id": "kFCmWEKdMA_Y"
+        "id": "kFCmWEKdMA_Y",
+        "outputId": "b4a568f5-5f73-49bf-ae21-35ede153273b",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 1000,
+          "referenced_widgets": [
+            "c5feb22ec5674243b90733e2ffb4c34c",
+            "b8a8b9c4e26e4073899f994651f038d7",
+            "7519b4f014ac483e993594f4fec3d75b",
+            "7917d7b06d8e46a897426d35536b18ca",
+            "34f80617f3044424b32d003c27ab5e21",
+            "c01bef63e78d4fd7b13d46dee4683fdc",
+            "79e6e1ee86914cfdb017758a83d3ca28",
+            "f3245896ff3c43af9046139fe92868f7",
+            "ed51b19dd65945da8459e8bb4b6cf7e0",
+            "c9d9cbc472434fa3bb67525e78f2bc48",
+            "e2c96cec21154d9e83df73b858493a1e"
+          ]
+        }
       },
       "outputs": [
         {
+          "output_type": "display_data",
```

Comment:
Optional, but I would clear output for cells like this - it makes the NB files big and users don't really care about download progress bars.

Reply:
Resolving so this can be merged